README.md

```diff
@@ -1,4 +1,11 @@
 # Cosmos Operator
+
+[![Project Status: Initial Release](https://img.shields.io/badge/repo%20status-active-green.svg?style=flat-square)](https://www.repostatus.org/#active)
+[![GoDoc](https://img.shields.io/badge/godoc-reference-blue?style=flat-square&logo=go)](https://godoc.org/github.com/strangelove-ventures/cosmos-operator)
```

Comment:
The redirect will probably continue to work indefinitely, but the URL should be pkg.go.dev instead of godoc.org now.

Reply:
Ah, thanks for that!